app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -1,10 +1,11 @@
 select {
   &#search_category {
-    background: url(asset-path("theme/dropdown_arrow.png")) no-repeat 95% center;
+    background: url(asset-path("theme/dropdown_arrow.png")) no-repeat 95% center transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered -moz-appearance, -webkit-appearance, appearance, background, background-size, border, box-shadow, font-weight, height, padding, width